source/authentication.rst

```diff
@@ -306,19 +306,47 @@ Authentication and Authorization with Direct Binding to LDAP
 
 Starting from release 4.2.5-5, |psmdb| supports |ldap-authorization|.
   
-This feature has been supported in |mongodb-e| since its version 3.4.
+This feature has been supported in |mongodb-e| since its version 3.4. 
   
 Note the following limitations of |ldap-authorization| in |psmdb|:
   
-- The |abbr.ldap| `connection pool and all related parameters are not supported <https://docs.mongodb.com/manual/core/security-ldap-external/#connection-pool>`_.
 - The `ldapTimeoutMS <https://docs.mongodb.com/manual/reference/program/mongoldap/#cmdoption-mongoldap-ldaptimeoutms>`_ parameter is ignored.
 - The `--ldapServers <https://docs.mongodb.com/manual/reference/program/mongoldap/#cmdoption-mongoldap-ldapservers>`_ option may only contain a single server (|mongodb-e| accepts a comma-separated list).
- 
+  
+As of version 4.2.10-11, |psmdb| supports LDAP referrals. For security reasons, referrals are disabled by default. Double-check that using referrals is safe before enabling them.
+
+To enable LDAP referrals, set the ``ldapFollowReferrals`` server parameter to ``1`` either using the ``setParameter`` command or editing the configuration file.
+
+.. code-block:: yaml
+
+   setParameter:
+      ldapFollowReferrals: 1
```

Comment:
despite '1' is a valid value, for boolean options, IMHO, it's preferable using true/false values, it's less confusing